legacy/src/api/assets/tokens.md

```diff
@@ -70,3 +70,110 @@ Token value may be set in multiple currencies and is the same for all tokens in
 | :----- | :-------------------- | :----------------------------------------------------------------------- |
 | 403    | TOKEN_ALREADY_CREATED | Token with supplied parameters already exists.                           |
 | 403    | LIVENESS_MISMATCH     | The account is test and the collection's liveness is main or vice versa. |
+
+## Create Token Collection
+
+{% reqspec %}
+  POST '/api/collections'
+  auth 'api-key'
+  body ({
+    "name": "Bread",
+		"accountId": "T3y6hogYA4d612BExypWYH",
+		"tokenExpiresAfter": {
+			"period": "months",
+			"duration": "1"
+		},
+		"maxValue": {
+			"currency": "NZD",
+			"amount": "400"
+		},
+		"type": "product",
+  })
+{% endreqspec %}
+
+{% h4 Fields %}
+
+|     Field      |  Type  |                                      Description                                       |
+| :------------- | :----- | :------------------------------------------------------------------------------------- |
+| name   | String | The display name of the collection. |
+| accountId | String | The account that will own the collection.                              |
+| tokenExpiresAfter   | String | The active duration of all tokens created from this collection. |
+| type | String | Supported values: <br> - `product`: tokens for this collection will be exchanged for one product. |
+| maxValue | {% dt Monetary %} {% opt %} | The maximum agreed value that any merchants will be settled for a token redemption. |
+| mediaUploadId | String {% opt %} | The id of the media upload image used for all tokens that are created from this collection. |
```

Comment:
Let's out of scope this. It's more work to implement this than we initially thought.

Reply:
Yep. will remove `mediaUploadId` field